ngx-http-concat.php

```diff
@@ -197,7 +197,7 @@ function ( $match ) {
 		// Only @charset rule are allowed before them.
 		if ( false !== strpos( $buf, '@import' ) ) {
 			$buf = preg_replace_callback(
-				'/(?P<pre_path>@import\s+(?:url\s*\()?[\'"\s]*)(?P<path>[^\'"\s](?:https?:\/\/.+\/?)?.+?)(?P<post_path>[\'"\s\)]*\W;)/i',
+				'/(?P<pre_path>@import\s?+(?:url\s*\()?[\'"\s]*)(?P<path>[^\'"\s](?:https?:\/\/.+\/?)?.+?)(?P<post_path>[\'"\s\)]*\W;)/i',
```

Comment:
The correct way to say zero or more space would be either `\s{0,} or `\s*`, I may have given you a misleading answer on the BB8, because `\s?` means zero or or one. 😬 

This would be closer to what we want, but a bit unsafe because it would match
`@importurl` - which wouldn't be valid.

```suggestion
				'/(?P<pre_path>@import\s{0,}(?:url\s*\()?[\'"\s]*)(?P<path>[^\'"\s](?:https?:\/\/.+\/?)?.+?)(?P<post_path>[\'"\s\)]*\W;)/i',
```

Reply:
cc @Automattic/vip-platform-cantina and @sjinks specifically for regex considerations;

Valid:
```
@import url(https://some.url);
@import "https://what.ever";
@import"https://what.ever";
```

Invalid: 
```
@importurl(...);